cloudflared/rootfs/etc/s6-overlay/scripts/10-cloudflared-config.sh

```diff
@@ -256,10 +259,12 @@ createConfig() {
 createDNS() {
```

Comment:
When looking at this method, I figured we might as well rename 'createCustomDNS' to 'createDNS' and use it all the time. Whether it is a custom_config or a config created by the add-on should not matter, right? @elcajon

Reply:
The difference is we use json for automatically generated config and YAML for custom config. We would need to streamline this first or simply combine both functions which wouldn't really reduce complexity.

Reply:
Yeah true, I forgot about that. Let's leave it as it was